source/index.html.erb

```diff
@@ -18,16 +18,14 @@ title: Index Page Example
     <div class="hero__content">
       <div class="grid-row">
         <div class="column-two-thirds">
-          <h1 class="hero__title">A hero unit</h1>
+          <h1 class="hero__title">Duis aute irure dolor in reprehenderit</h1>
```

Comment:
Made this longer so that it wraps onto two lines, which it does in many of the prototype product pages.